src/price-formatting/format.js

```diff
@@ -199,7 +199,10 @@ function formatPriceParts(numberFormatting, value, decimals, formatFlags, numera
 
     if (isNegative) {
         parts.Post = numberFormatting.negativePost;
-        parts.Pre = numberFormatting.negativePre;
+
+        // Infinitesimally small negative value is rounded to 0, in which case Pre should not be '-'
+        // as '-0' makes no sense, hence the below check
+        parts.Pre = (parts.First === '0' && !parts.Pips) ? '' : numberFormatting.negativePre;

```

Comment:
To be thorough, should we check decipips too?

Reply:
I discussed this with Mayank, decipips won't come untill and unless there is a value for pips. The decipips check would be redundant. Hence skipping it.

Reply:
@abpasaxo I think mayank is wrong .. it’s probably never going to happen but let’s code for it in case

Reply:
Added check for decipips.